common/constants/default.go

```diff
@@ -0,0 +1,19 @@
+package constants
+
+// Resources
+const (
+	ResourceTypeSecret = "secret"
+
+	MY_NODE_NAME    = "MY_NODE_NAME"
+
+	NamespaceSystem = "kubeedge"
+
+	ServerDefaultACLDirectory = "/etc/kubeedge/edgemesh/server/acls"
+	AgentDefaultACLDirectory = "/etc/kubeedge/edgemesh/agent/acls"
+	ServerDefaultKeyFile    = ServerDefaultACLDirectory + "/server.key"
```

Comment:
It is recommended to use `path.Join`

Reply:
ok, good, i will modify it

Reply:
path.join return a string not a constant